numba/unicode.py

```diff
@@ -382,6 +382,32 @@ def _find(substr, s):
     return -1
 
 
+#The following function handles special case for rfind where substring is empty
+@njit
```

Comment:
If this function really needs to be external to the implementation, then `@register_jitable` should be used in place of `@njit`.

Reply:
Yes right. I recently got aware of register_jitable's usage so I understand what you're saying. I'll change that.